docs/clean-architecture.md

```diff
@@ -416,7 +416,8 @@ These architectures are:
     - Independent of the database.
     - Independent of any external agency.
 
-*******Please put a clean architecture image here********
+![Clean architecture image](https://github.com/Majdi-Zlitni/Book-notes/assets/60318289/755fde8a-bc53-4571-afac-a7bc37c9eaf0)
```

Comment:
I am thanking you right now for your reminder about this. I don't know how I forgot about it.   
Adding an image like that is not the best thing. It's better to make the image a local asset inside the project to prevent any problems (it's not like the cover images appear at the top where if they broke the user won't have any problem understanding the content of the project).  
Therefore I am going to close this PR but I promise I will use your image.